config/new_parameters.yml

```diff
@@ -0,0 +1,119 @@
+#########################################################
```

Comment:
Could we name this file differently? What's it for -- calibration in Manchester?